airflow/dags/fda_excluded/dag.py

```diff
@@ -5,26 +5,30 @@
 
 from common_dag_tasks import  extract, transform, get_ordered_sql_tasks, get_ds_folder
 from sagerx import read_sql_file
+from airflow.decorators import dag,task
 from airflow.providers.postgres.operators.postgres import PostgresOperator
 
 
 dag_id = "fda_excluded"
 
 dag = create_dag(
     dag_id=dag_id,
-    schedule= "0 4 * * *",  # run at 4am every day
-    start_date=pendulum.yesterday(),
-    catchup=False,
+    schedule= "30 4 * * *",  # run at 4:30am every day
     max_active_runs=1,
     concurrency=2,
 )
 
+@task(task_id="target_dag")
```

Comment:
I dont understand the purpose of this task

Reply:
I actually did not add this task - at least if I did, I also have no idea what it's really for other than to add some additional logging.

Reply:
If its not something you added then i would remove it.